File: Examples/SimpleApp/app/src/main/java/com/optimove/sdk/demo/MainActivity.java
package com.optimove.sdk.demo;

import android.os.Bundle;
import android.widget.Toast;

import com.optimove.sdk.optimove_sdk.main.Optimove;
import com.optimove.sdk.optimove_sdk.main.constants.SdkRequiredPermission;

import androidx.appcompat.app.AppCompatActivity;
import androidx.viewpager.widget.ViewPager;

public class MainActivity extends AppCompatActivity {

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_main);
    ViewPager viewPager = findViewById(R.id.mainViewPager);
    MainPagerAdapter mainPagerAdapter = new MainPagerAdapter(getSupportFragmentManager());
    viewPager.setAdapter(mainPagerAdapter);
    if (BuildConfig.DEBUG) {
      Optimove.getInstance().startTestMode(success -> Toast.makeText(this, "Test Mode Started", Toast.LENGTH_SHORT).show());
    }
  }
}

File: Examples/Kotlin/app/src/main/java/com/optimove/sdk/demo/MainActivity.kt
package com.optimove.sdk.demo

import android.os.Bundle
import android.widget.Toast

import com.optimove.sdk.optimove_sdk.main.Optimove
import com.optimove.sdk.optimove_sdk.main.constants.SdkRequiredPermission

import androidx.appcompat.app.AppCompatActivity
import androidx.viewpager.widget.ViewPager

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val viewPager = findViewById<ViewPager>(R.id.mainViewPager)
        val mainPagerAdapter = MainPagerAdapter(supportFragmentManager)
        viewPager.adapter = mainPagerAdapter
        if (BuildConfig.DEBUG) {
            Optimove.getInstance().startTestMode { success -> Toast.makeText(this, "Test Mode Started", Toast.LENGTH_SHORT).show() }
        }
    }
}
